aip/0158.md

```diff
@@ -74,6 +74,8 @@ message ListBooksResponse {
 - Response messages for collections **should** define a
   `string next_page_token` field, providing the user with a page token that may
   be used to retrieve the next page.
+  - The field being paginated over **should** be the first field in the message
```

Comment:
```suggestion
  - The field containing pagination results **should** be the first field in the
    message
```

I don't know how to do multi-line comments... sorry about this...

Reply:
Eh, I can `git pull` and fix it. :)